api/index.js

```diff
@@ -33,6 +42,11 @@ const start = () => {
   };
 
   app.use(cors(corsOptions));
+  
+  app.use(compression({
+    filter: shouldCompress,
+    threshold: 0
```

Comment:
http://expressjs.com/en/resources/middleware/compression.html#threshold do we need this? Should we just leave it to default?